src/handlers/localstorage/mmkvStorageBackend.ts

```diff
@@ -0,0 +1,29 @@
+import { MMKV } from 'react-native-mmkv';
+import { STORAGE_IDS } from '../../model/mmkv';
+
+const mmkvLocalStorage = new MMKV({
+  id: STORAGE_IDS.LOCAL_STORAGE_ADAPTER,
```

Comment:
I think at this point we should unify buttons for clearing mmkv and local storage since I cannot think of the obvious way of presenting those buttons in a correct way. And I assume we don't need to separate them anyway. 

Do you expect we don't need those other methods anyway in this adapter? https://react-native-async-storage.github.io/async-storage/docs/api

Should